docs/DeveloperGuide.adoc

```diff
@@ -279,51 +279,203 @@ Refer to the guide <<DevOps#, here>>.
+
+|`* *` |family member of a diabetic |prioritize my tasks |be immediately contactable if my family member has an emergency situation that requires urgent medical attention
+
+|`* *` |diabetic |have a customisable app with avatars and different backgrounds |enjoy a personalised experience
+
+|`* *` |lazy and obese individual |be motivated constantly to exercise |stop procrastinating
+
+|`* *` |forgetful diabetics patient |have a record of my doctors' advice for each medical appointment and prescription directions |better understand the steps that I can take to improve my condition until the next consultation
+
+|`* *` |achievement-oriented diabetic | view the achievements and progress I have made on food intake |remain motivated to keep my streak on good habits going
+
+|`* *` |paranoid diabetic who values privacy |secure/encrypt my health data and other private contact details |protect my data
+
+|`* *` |diabetic patient with a family |have a user-friendly app that helps me manage my medical data and appointments on my own |free the burden I have on my family
+
+|`* *` |diabetic patient with a family |have a user-friendly app with natural commands that helps me manage my medical data and appointments on my own |free the burden I have on my family
+
+|`*` |diabetic patient in a community of diabetic patients |have a standardised means of comparing our activities via a social network | learn from my peers, encourage and be encouraged through this difficult journey.
 
-_{More to be added}_
+|`*` |careless user  |undo my most recent actions |easily make necessary amendments and input the correct commands
+
+|`*` |a diabetic patient who has many medical receipts - and is not very good at mathematics |have a simple calculator that is always easily accessible |instantly calculate all my medical costs when needed
+
+|`*` |an obese working adult at high risk of diabetes |start monitoring my diet |minimise my risk of having diabetes
+
+|`*` |medical consultant |export my patient's health data  |save my time
+|=======================================================================
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `Sugar Mummy` and the *Actor* is the `user`, unless specified otherwise)
+
+=== Use case: Add blood sugar record
+*MSS*
+
+1.  User requests to add a blood sugar record
+2.  System adds the blood sugar record
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. The record is incomplete or passed invalid arguments.
++
+[none]
+** 1a1. System shows an error message.
++
+Use case resumes at step 1.
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Schedule a medical appointment
+*MSS*
+
+1.  User requests to add a medical appointment
```

Comment:
Unclear whether user is requesting or adding a medical appointment. Can be rephrased to "User adds a medical appointment".